lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -5,18 +5,14 @@
 
 foam.CLASS({
   package: 'org.chromium.webidl',
-  name: 'LocalGitIDLFileDAO',
-  extends: 'foam.dao.ProxyDAO',
+  name: 'LocalGitRunner',
+  extends: 'foam.box.Runnable',
```

Comment:
My main concern is that it may be confusing or difficult to remember which box is which if one has the generic name of `outputBox`.